### DISCUSSION TITLE
Add setting to disable kiai star effect in gameplay

### DISCUSSION THREAD

**Discussion by M1N0RM1N3R** [ppy/osu#29830] [Ideas] [answered]
A simple toggle--"Show star effects in kiai time"--under the Gameplay category (most likely) is all we ask.

Look. I get it. You're trying to get on par with stable...
The reintroduction of the kiai star effect in gameplay is all well and good for casual players, but for people like me (and everyone who set `star2.png` to a transparent pixel in their stable skin), it's way too distracting to have during gameplay.
I love the look and feel of the argon pro skin included with the game, and have been using it (and its non-pro counterpart) nearly exclusively ever since I first downloaded lazer. I tried exporting my (layout-adjusted) copy, extracting, adding a `star2.png` pixel, recompressing, renaming, and importing--the fountains are still there.
For the time being, I suppose I could rebuild the skin from the osu-resources repo, but most people wouldn't know to do that.
Welp, time to get to work...

**Comment by Tenexxt** [comment]
They are working on also adding combo bursts from stable (the characters popping out from the sides when the combo is high) so I think they will give an option to disable the kiai stars with that, since in the Dev video it was shown that combo bursts will be a toggle.

**Comment by ChenMerami** [comment]
I hate that there is no option to disable it in argon "pro"

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/30894
